libraries/AP_GPS/AP_GPS_SBP2.cpp

```diff
@@ -0,0 +1,504 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+//
+//  Swift Navigation SBP GPS driver for ArduPilot.
+//	Code by Niels Joubert
+//
+//  Swift Binary Protocol format: http://docs.swift-nav.com/
+//
+
+#include "AP_GPS.h"
+#include "AP_GPS_SBP2.h"
+#include <DataFlash/DataFlash.h>
+
+extern const AP_HAL::HAL& hal;
+
+#define SBP_DEBUGGING 1
+#define SBP_DEBUGGING_SUPERVERBOSE 1
+#define SBP_HW_LOGGING 1
```

Comment:
Yes, I think that leaving something called debug turned on isn't very appropriate. If it is always on, you can even remove the macro.